python/controllers/create_file_group_controller.py

```diff
@@ -0,0 +1,140 @@
+import os
+import azure.batch_extensions as batch
+
+from server.app import app
+from jsonrpc import JsonRpcErrorCodes, error
+from msrestazure.azure_active_directory import AdalAuthentication
+
+SUBDIR_FILTER = "**\\*"
```

Comment:
yeah the path.join makes a \\ because you are on windows but on linux it will make a /